examples/lammps/simple_lammps/trajectory_continuer.py

```diff
@@ -0,0 +1,137 @@
+"""
+Example of a user-provided trivial analysis tool.
+
+Set up a SCALE-MS compatible operation implementation and export it as the
+module function `trajectory_continuer`.
+"""
+
+#import scalems
+import pathlib
+from pathlib import Path
+import MDAnalysis
```

Comment:
Do we expect MDAnalysis to be a common dependency? We can add it to our core test environments.

Hmm... #90 doesn't mention Python package dependencies in the execution environment, and I am not seeing the topic in other issues either. I created #141 to put that onto the road map more explicitly.